packages/backend-data/src/convert_schema.ts

```diff
@@ -61,3 +80,54 @@ export const convertSchemaToCDK = (
     provisionStrategy,
   });
 };
+
+/**
+ * Given an input list of CDK Graphql Def interface schemas, produce the relevant CDK Graphql Def interface
+ * @param schemas the cdk graphql definition interfaces to combine
+ * @returns the cdk graphql definition interface
+ */
+export const combineCDKSchemas = (
+  schemas: IAmplifyDataDefinition[]
+): IAmplifyDataDefinition => {
+  return AmplifyDataDefinition.combine(schemas);
+};
+
+/**
+ * Given the generated rds builder database configuration, convert it into the DataSource strategy
+ * @param configuration the database configuration from `data-schema`
+ * @returns the data strategy needed to configure the data source
+ */
+const convertDatabaseConfigurationToDataSourceStrategy = (
+  configuration: DataSourceConfiguration
+): ModelDataSourceStrategy => {
+  // DO NOT EDIT THE FOLLOWING VALUES, UPDATES TO DB TYPE OR STRATEGY WILL RESULT IN DB REPROVISIONING
+  const defaultDbType = 'DYNAMODB';
+  const defaultProvisionStrategy = 'AMPLIFY_TABLE';
+
+  const dbEngine = configuration.engine;
+
+  if (dbEngine === 'dynamodb') {
+    return {
+      dbType: defaultDbType,
+      provisionStrategy: defaultProvisionStrategy,
+    };
+  }
+
+  const dbType = <Uppercase<typeof configuration.engine>>dbEngine.toUpperCase();
```

Comment:
Here we're configuring to screaming snake case which we decided not to expose to customers previously. We should probably plumb the lowercase versions through as far as possible instead of translating.